### PR TITLE
added DList_RemoveTailList

### PR DESCRIPTION
**Pull request by anporumb**

### CODE REVIEW

File: devdoc/doublylinkedlist_requirements.md [hunk 1]
```diff
@@ -122,6 +122,19 @@ extern PDLIST_ENTRY DList_RemoveHeadList(PDLIST_ENTRY listHead);
 
 Note: The Flink & Blink of the returned PDLIST_ENTRY shall be undefined.
 
+### DList_RemoveHeadList
+```c
+extern PDLIST_ENTRY DList_RemoveTailList(PDLIST_ENTRY listHead);
```
Comment (dcristoloveanu) [inline devdoc/doublylinkedlist_requirements.md:127]: >extern [](http://example.com/codeflow?start=0&length=6)

Likely not needed

**Comment by dcristoloveanu** [comment]
I find it intriguing that the PR name is DList_RemoveHeadList but the contents are DList_RemoveTailList :-).
Keeping people on their toes!

File: devdoc/doublylinkedlist_requirements.md (hunk 1 above)
Comment (anporumb) [reply devdoc/doublylinkedlist_requirements.md:127]: Removed here and everywhere

**Comment by anporumb** [comment]
Me too!

---
In reply to: [1238796339](https://github.com/Azure/c-util/pull/206#issuecomment-1238796339) [](http://example.com/codeflow?ancestors=1238796339,1238796339)

**Pull request merged**